src/Features/LanguageServer/Protocol/Features/UnifiedSuggestions/UnifiedSuggestedActions/IFixAllCodeRefactoringSuggestedAction.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+
+namespace Microsoft.CodeAnalysis.UnifiedSuggestions.UnifiedSuggestedActions
+{
+    /// <summary>
+    /// Common interface used by both local Roslyn and LSP to implement
+    /// their specific versions of FixAllCodeRefactoringSuggestedAction.
+    /// </summary>
+    internal interface IFixAllCodeRefactoringSuggestedAction
+    {
+        CodeAction OriginalCodeAction { get; }
```

Comment:
curius why this needs to be exposed through the interface.

Reply:
Matching this: https://github.com/dotnet/roslyn/blob/e1629052738683dac234a4d75bb1d1c442c46d7d/src/Features/LanguageServer/Protocol/Features/UnifiedSuggestions/UnifiedSuggestedActions/IFixAllSuggestedAction.cs#L18

It seems to be passed around elsewhere.

Reply:
Seems to be eventually used here for telemetry: https://github.com/dotnet/roslyn/blob/e1629052738683dac234a4d75bb1d1c442c46d7d/src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/FixAllSuggestedAction.cs#L61